dropshot/src/api_description.rs

```diff
@@ -33,7 +34,7 @@ use std::collections::HashSet;
 #[derive(Debug)]
 pub struct ApiEndpoint<Context: ServerContext> {
     pub operation_id: String,
-    pub handler: Box<dyn RouteHandler<Context>>,
+    pub handler: Arc<dyn RouteHandler<Context>>,
```

Comment:
Why did this change to `Arc`?

Reply:
Got it -- in the `spawn` case, we need the handler to be `'static`, so it can't be a reference back to something.